src/content/configuration/dev-server.md

```diff
@@ -132,7 +132,7 @@ module.exports = {
 To use this option with the CLI pass the `--allowed-hosts` option a comma-delimited string.
 
 ```bash
-npx webpack serve --entry ./entry/file --output-path ./output/path --allowed-hosts .host.com,host2.com
+npx webpack serve --entry ./entry/file --output-path ./output/path --allowed-hosts .host.com --allowed-hosts host2.com
```

Comment:
![image](https://user-images.githubusercontent.com/1091472/108501000-98b7de80-72eb-11eb-9756-97d01840bfd6.png)
Anything wrong?
```
npx webpack --version
webpack 5.23.0
webpack-cli 4.5.0
webpack-dev-server 4.0.0-beta.0
```

Reply:
Use webpack-dev-server v3.11.2

Reply:
Many options removed in v4, I will update after a major release.

Reply:
So it's not supported on `webpack-dev-server` v4? Which is weird to me.

Reply:
We have `--firewall` in v4.

Reply:
![Screenshot at 2021-02-19 17-36-09](https://user-images.githubusercontent.com/46647141/108502574-027abd00-72d9-11eb-84c3-bf846cd8f59e.png)
![Screenshot at 2021-02-19 17-35-59](https://user-images.githubusercontent.com/46647141/108502594-08709e00-72d9-11eb-9832-c46a17faec3a.png)

Reply:
yeah, I've tested it.